core/websock.js

```diff
@@ -95,6 +95,13 @@ export default class Websock {
                this._rQ[this._rQi++];
     }
 
+    rQshift32L() {
```

Comment:
I'm a bit sceptical adding more wrappers here. And this would be incomplete as you would also need 16-bit and peek functions. And there is just the one call to this. I think it is better to swap the result in the handler instead (ntohl() style).

Reply:
I think it's better to get little endian value in high performance websock. If it is not appropriate, i can convert it in the handler.

Reply:
There is no performance requirement here. So put it in the handler.